setup.py

```diff
@@ -96,10 +96,10 @@ def run_setup():
         license='BSD',
         classifiers=classifiers,
         install_requires=[
-            'networkx >= 1.8, <= 1.10',
+            'networkx >= 2.0, <= 2.4',
             'numpy >= 1.7',
-            'omega >= 0.0.9, < 0.1.0',
-            'ply >= 3.4',
+            'omega >= 0.1.2',
```

Comment:
I suggest pinning to `omega == 0.2.0` or in a range `omega >= 0.1.2, <= 0.2.0`, to allow for changes in the API of `omega`.

Reply:
@johnyf I agree. I am preparing several commits to add to this PR from my temporary branch `tmp-debug-travisci`.

Reply:
Planning to accept this change as is, and let @slivingston fix this in `tmp-debug-travisci`.

Reply:
Confirmed. I will open a separate PR with my changes and those from the branch `completed_update_to_networkx_2` by @johnyf